.gitignore

```diff
@@ -1,6 +1,9 @@
 # Compiled class file
 *.class
 
+# Hide files not needed for migration pull request
+src/main/java/io/jenkins/plugins/audit/*
```

Comment:
This does not belong in the master branch.

Reply:
just corrected, forgot to remove it from the repo